portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -17,6 +17,11 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.sps.input.Comments;
 import java.io.IOException;
 import java.util.*;
```

Comment:
I didn't notice this earlier either.

import just the libraries you need.
#import java.util.ArrayList;
#import java.util.List;

Reply:
Done